src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/messages.html.twig

```diff
@@ -0,0 +1,65 @@
+{% extends '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% import _self as helper %}
+
+{% block menu %}
+<span class="label">
+    <span class="icon">{{ include('@WebProfiler/Icon/messages.svg') }}</span>
+    <strong>Messages</strong>
+
+    {% if collector.messages|length > 0 %}
+        <span class="count">
+            <span>{{ collector.messages|length }}</span>
+        </span>
+    {% endif %}
+</span>
+{% endblock %}
+
+{% block panel %}
+    <h2>Messages</h2>
+
+    {% if collector.messages is empty %}
+    <p>No messages</p>
+    {% else %}
+    <table>
+        <thead>
+        <tr>
+            <th>Message</th>
+            <th>Result</th>
+        </tr>
+        </thead>
+        <tbody>
+        {% for message in collector.messages %}
+        <tr>
+            <td>{{ message.message.type }}</td>
+            <td>
+                {% if message.result.type is defined %}
+                    {{ message.result.type }}
+                {% endif %}
+
+                {% if message.exception.type is defined %}
+                    {{ message.exception.type }}
+                {% endif %}
+            </td>
+        </tr>
+        {% endfor %}
+        </tbody>
+    </table>
+    {% endif %}
+{% endblock %}
+
+{% block toolbar %}
+    {% set color_code = 'normal' %}
+    {% set message_count = 0 %}
+    {% set icon %}
+        {% if profiler_markup_version == 1 %}
```

Comment:
do we still support this? :|

Reply:
Good question I don't know how to answer. I reused another template so kept this.

Reply:
I don't see anything like this in the Symfony's codebase. In any case, version 1 was pre-2.8, so no need to support it here.